web/yo/app/scripts/controllers/gene.js

```diff
@@ -3311,13 +3311,23 @@ angular.module('oncokbApp')
         var geneName = data.geneName;
 
         $scope.saveCallback = function(newTreatmentName, oldContent){
-            $scope.treatmentRef.name = newTreatmentName;
             var name_uuid = $scope.treatmentRef.name_uuid;
-            mainUtils.setUUIDInReview(name_uuid);
-            var therapyObject = {
-                'name': newTreatmentName,
-                'status': 'latest'
-            };
+            if(!_.isEmpty($scope.treatmentRef.name)) {
```

Comment:
Only update the review content when it's brand new treatment?

Reply:
Update the review content when there is old treatment name. I move the old treatment name to review content. Then I update treatment name.